bundle/manifests/rhdh.redhat.com_orchestrators.yaml

```diff
@@ -53,6 +53,15 @@ spec:
                       operator is deployed and is watching for argoapp CR instances
                     type: string
                 type: object
+              networkPolicy:
```

Comment:
This one does not exist in the helm chart values.yaml file, and should not be here. Is the file config/crd/bases/rhdh.redhat.com_orchestrators.yaml.

Reply:
It's in the `values.yaml` file:
https://github.com/rhdhorchestrator/orchestrator-helm-operator/blob/6a40600a451c59ba8f44744095a6db4133955d98/helm-charts/orchestrator/values.yaml#L111

Reply:
ok